packages/web/app/forms/ProcedureForm/ProcedureFormModals.jsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TranslatedText } from '../../components';
+import { ConfirmModal } from '../../components/ConfirmModal';
+
+const StyledConfirmModal = styled(ConfirmModal)`
+  &.MuiDialog-root {
+    z-index: 100;
+  }
+`;
+
+export const UnSavedChangesModal = ({ open, onCancel, onConfirm }) => (
+  <StyledConfirmModal
+    title={
+      <TranslatedText stringId="procedure.modal.unsavedChanges.title" fallback="Unsaved changes" />
```

Comment:
You've gone for one word "unsaved" here, so yeah I reckon update all the other casing to match in variable and component names